content/en/APIUseCases/get-findings.md

```diff
@@ -0,0 +1,172 @@
+
+Next, you can use the API token to authorize access to the Cobalt API. Take
+the API token that you [generated](#step-1-create-an-api-token-in-the-cobalt-ui). Substitute that value for `{{% api-token %}}`:
+
+```bash
+curl -X GET "https://api.cobalt.io/orgs" \
+     -H "Accept: application/vnd.cobalt.v2+json" \
+     -H "Authorization: Bearer {{% api-token %}}" \
+```
+
+{{%expand "Click to view a sample response." %}}
+You should see output similar to:
+
+```json
+{
+  "data": [
+    {
+      "resource": {
+        "id": "some_id",
+        "name": "Name of your organization",
+        "token": "{{% org-token %}}"
+      },
+      "links": {
+        "ui": {
+          "url": "https://api.cobalt.io/links/link-to-web-app"
+        }
+      }
+    }
+  ],
+  "pagination": {
+    "next_page": "/orgs?cursor=a1b2c3d4",
+    "prev_page": "/orgs?cursor=4d3c2b1a"
+  }
+}
+```
+{{% /expand %}}
+</br>
+  
+From the output, save the value for `token` as your organization token.
+In our API documentation, you'll see this as `{{% org-token %}}`.
+
+For more information, see our API reference documentation on the
+[organizations](https://docs.cobalt.io/v2/#organizations) `orgs` endpoint.
+
+## Step 3: Get All Findings
+
+Now that you have the following information:
+
+- `{{% api-token %}}`
+- `{{% org-token %}}`
+
+You can retrieve all [findings](/platform-deep-dive/pentests/findings/) with the following REST call:
+
+```bash
+curl -X GET "https://api.cobalt.io/findings" \
+  -H "Accept: application/vnd.cobalt.v2+json" \
+  -H "Authorization: Bearer {{% api-token %}}" \
+  -H "X-Org-Token: {{% org-token %}}"
+```
+
+For more information on each parameter, see our API reference documentation on
+how to [get all findings](https://docs.cobalt.io/v2/#get-all-findings).
+
+If the command is successful, you'll see
+
+| Message    | Meaning          |
+|------------|------------------|
+| 200 OK | Findings retrieved    |
+
+For a list of error codes, see the [Errors](https://docs.cobalt.io/v2/#errors)
+section of our API reference.
+
+## Step 4: Import Findings to Google Sheets
+
+Now that you have findings data in JSON format, you can import it to the spreadsheet software of your choice.
+
+As an example, let's import findings to Google Sheets.
+
+1. In Google Sheets, select **Extensions** > **Apps Script**.
+1. Use the [Class UrlFetchApp](https://developers.google.com/apps-script/reference/url-fetch/url-fetch-app) service to import raw findings data. Refer to the [Class SpreadsheetApp](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet-app) reference for details.<br>
+1. Run the following script to download and import Cobalt findings. Replace `{{% api-token %}}` and `{{% org-token %}}` with your values.<br>
+
+    ```js
+    function FETCH_DATA(pentest) {
+      var url = 'https://api.cobalt.io/findings?pentest='+pentest; 
+      var params = { headers: {
+        'Accept': 'application/vnd.cobalt.v1+json',
```

Comment:
@ana-dashuk-cobalt we should make sure we're making this work for our V2 findings endpoint, not V1 which you can see in this header

Reply:
@nktrk Updated.